packages/http-client-csharp/eng/scripts/Generate.ps1

```diff
@@ -15,32 +15,32 @@ if (-not $LaunchOnly) {
 
     if ($null -eq $filter -or $filter -eq "Sample-TypeSpec") {
 
-        Write-Host "Building logging plugin" -ForegroundColor Cyan
-        $pluginDir = Join-Path $packageRoot '..' '..' 'docs' 'samples' 'client' 'csharp' 'plugins' 'logging' 'Logging.Plugin' 'src'
-        Invoke "dotnet build" $pluginDir
-
-        $sampleDir = Join-Path $packageRoot '..' '..' 'docs' 'samples' 'client' 'csharp' 'SampleService'
-
-        Write-Host "Installing SampleTypeSpec plugins" -ForegroundColor Cyan
-
-        Invoke "npm install --force" $sampleDir
-
-        Write-Host "Generating SampleTypeSpec using plugins" -ForegroundColor Cyan
-
-        Invoke "npx tsp compile . --trace @typespec/http-client-csharp" $sampleDir
-
-        # exit if the generation failed
-        if ($LASTEXITCODE -ne 0) {
-          exit $LASTEXITCODE
-        }
-
-        Write-Host "Building SampleTypeSpec plugin library" -ForegroundColor Cyan
-        Invoke "dotnet build $sampleDir/SampleClient/src/SampleTypeSpec.csproj"
-
-        # exit if the generation failed
-        if ($LASTEXITCODE -ne 0) {
-          exit $LASTEXITCODE
-        }
+#        Write-Host "Building logging plugin" -ForegroundColor Cyan
```

Comment:
Temporarily commenting out as we are adding a string nextlink scenario that won't build with old generator version.